tsl/test/sql/dist_compression.sql

```diff
@@ -491,6 +491,35 @@ SELECT add_continuous_aggregate_policy('test_recomp_int_cont_view', NULL, BIGINT
 CALL refresh_continuous_aggregate('test_recomp_int_cont_view', NULL, NULL);
 SELECT * FROM test_recomp_int ORDER BY 1;
 SELECT * FROM test_recomp_int_cont_view ORDER BY 1;
+
+--TEST cagg triggers work on distributed hypertables when we insert into 
+-- compressed chunks.
+SELECT compress_chunk(chunk, true) FROM
+( SELECT chunk FROM show_chunks('test_recomp_int') AS chunk ORDER BY chunk )q;
```

Comment:
```suggestion
SELECT compress_chunk(chunk, true)
FROM show_chunks('test_recomp_int') AS chunk
ORDER BY chunk;
```

Reply:
Just a minor readability SQL reworking (up to you to accept)... approved anyway

Reply:
modified the query to incorporate Sven's suggestion.